CSC322-Grinnell/mayflower-directory
# Combine directory with pictures page

Resolves #175, resolves #156, resolves #153.

# Changes

- There is no longer a pictures page.
- Instead, pictures are shown on the directory page and are now searchable.
- The directory page is printable.
- Pictures are now cropped to maintain aspect ratio.
- Improved contrast of names under pictures.

# Testing

1. If you don't have any profiles yet, add them with `rails db:dummy_seed`
2. Start the server
3. Go to the directory page
4. Search for a name that should appear in the profile list such as "Alex"
5. Search for a name that shouldn't appear in the profile list such as "Foo"
6. Try narrowing by neighborhood
7. Combine neighborhood with text search
8. Try to print the page and make sure the print preview looks good (Ctrl-P or Command-P).  The search bar shouldn't show up when printing, and the profiles shouldn't be cut off halfway down a page
